src/app/views/admin-views/listof-category/listof-category-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { ListofCategoryComponent } from './listof-category.component';
+
+
+const routes: Routes = [
+  {
+    path:'',
+    component:ListofCategoryComponent
+  }
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule]
+})
+export class ListofCategoryRoutingModule { }
```

Comment:
Change module to CategoryListRoutingModule

Reply:
Resolved.